elm_deps_check.py

```diff
@@ -4,61 +4,144 @@
 import sys
 import json
 import argparse
-
-
-def have_matching_versions(top_level_file, spec_file, is_exact=False, quiet=True):
-    """ first file should be the top level elm-package exact-dependencies.json
-        second file should be the spec file
+import textwrap
+
+
+class PackageNotFound(object):
+    friendly = textwrap.dedent('''\
+    Package {package_name} was a dependency in the reference package file
+    ({reference_name}) but was not in the candidate package file
+    ({candidate_name}).
+
+    You can probably fix this error by adding {package_name} to the
+    dependencies in {candidate_name}.\
+    ''')
```

Comment:
💭 Newlines will be preserved here, so — unless something else is reformatting paragraphs later on — this may render poorly if the paths are long. It's not as convenient, but the way I might deal with this is:

```python
class PackageNotFound(object):
    friendly = (
        "Package {package_name} was a dependency in the reference package "
        "file ({reference_name}) but was not in the candidate package file "
        "({candidate_name})."
        "\n\n"
        "You can probably fix this error by adding {package_name} to the "
        "dependencies in {candidate_name}."
    )
```

Reply:
fixed in b537f6e